flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -1408,28 +1418,40 @@ mlir::Value Fortran::lower::createSomeExpression(
     mlir::Location loc, Fortran::lower::AbstractConverter &converter,
     const Fortran::evaluate::Expr<Fortran::evaluate::SomeType> &expr,
     Fortran::lower::SymMap &symMap) {
-  return ExprLowering{loc, converter, expr, symMap}.genval();
+  Fortran::lower::ExpressionContext bogon;
+  return ExprLowering{loc, converter, symMap, bogon}.genValue(expr);
 }
 
 fir::ExtendedValue Fortran::lower::createSomeExtendedExpression(
     mlir::Location loc, Fortran::lower::AbstractConverter &converter,
     const Fortran::evaluate::Expr<Fortran::evaluate::SomeType> &expr,
-    Fortran::lower::SymMap &symMap, llvm::ArrayRef<mlir::Value> lcvs) {
-  return ExprLowering{loc, converter, expr, symMap, lcvs}.genExtVal();
+    Fortran::lower::SymMap &symMap,
+    const Fortran::lower::ExpressionContext &context) {
+  return ExprLowering{loc, converter, symMap, context}.genExtValue(expr);
 }
 
 mlir::Value Fortran::lower::createSomeAddress(
     mlir::Location loc, Fortran::lower::AbstractConverter &converter,
     const Fortran::evaluate::Expr<Fortran::evaluate::SomeType> &expr,
     Fortran::lower::SymMap &symMap) {
-  return ExprLowering{loc, converter, expr, symMap}.gen();
+  Fortran::lower::ExpressionContext bogon;
```

Comment:
bogon?  Did the Romulans invent a new galactic particle beam weapon? :)

Reply:
Maybe I should've called it "bigDummy"?